src/gui/executionlog.cpp

```diff
@@ -28,15 +28,21 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include "executionlog.h"
+
+#include "guiiconprovider.h"
```

Comment:
OK ,but I still think that it is better to place them in the following order:
1. corresponding header file
2. app includes
3. Qt include
4. STL and other standard includes

Why? Because standard library includes are of less importance, everybody knows what is inside , but what is much more important is app modules inter-dependencies, showed by  app includes.